.github/workflows/ci.yml

```diff
@@ -9,6 +9,22 @@ on:
     - cron: '38 0 * * *' # every day at randint(0, 60) minutes after midnight
 
 jobs:
+  test-plugin-json:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get install jq
+
+      - name: plugin_json_to_cloud.sh should not change src.plugin.json
+        run: |
+          jq --sort-keys . src/plugin.json > /tmp/before.json
+          bash utils/plugin_json_to_cloud.sh
+          jq --sort-keys . src/plugin.json > /tmp/after.json
+          diff /tmp/before.json /tmp/after.json
```

Comment:
Can we actually test something here? Should we? I'm not sure

Reply:
the problem is: we need both scripts to_cloud and to_unsigned in order to support the two yarn scripts `build` and `build:cloud`. otherwise one of them would not work after executing the other.
but one script (to_cloud or to_unsigned) should not modify the state of `src/plugin.json` at all, because in the checkout of the repo, already contains the default.
the ugly thing is: `src/plugin.json` is prettified, and jq and prettifier have different styles. that's why i have to use jq for creating `/tmp/before.json`, otherwise we could copy that.
the last command will fail if both files differ, i tested it in the ci, by changing the default of `src/plugin.json` but not adapting this test: https://github.com/Checkmk/grafana-checkmk-datasource/commit/13686e9964e2ae8c9cd0a5ae7e4f6e1c6ae58f3c
(in other words: there is code duplication between `src/plugin.json` and `plugin_json_to_cloud.sh` and this test makes sure they stay in sync <- maybe we can add this as a comment to clear things up?)